log4j-osgi-test/src/test/java/org/apache/logging/log4j/osgi/tests/CoreOsgiTest.java

```diff
@@ -62,6 +63,19 @@ public Option[] config() {
                 junitBundles());
     }
 
+    @ProbeBuilder
+    public TestProbeBuilder probeConfiguration(final TestProbeBuilder builder) {
+        // Register `Log4jPlugins` manually with the Service Loader Mediator
+        builder.setHeader(
+                Constants.PROVIDE_CAPABILITY,
+                "osgi.serviceloader;osgi.serviceloader=\"org.apache.logging.log4j.plugins.model.PluginService\";"
+                        + "register:=\"org.apache.logging.log4j.osgi.tests.plugins.Log4jPlugins\"");
+        builder.setHeader(
+                Constants.REQUIRE_CAPABILITY,
+                "osgi.extender;filter:=\"(&(osgi.extender=osgi.serviceloader.registrar)(version>=1.0)(!(version>=2.0)))\"");
+        return builder;
+    }
```

Comment:
Since PAX Exam generates its own manifest, these properties need to be added by hand.